presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/RandomizeNullKeyInOuterJoin.java

```diff
@@ -386,6 +384,15 @@ public PlanNode visitJoin(JoinNode joinNode, RewriteContext<Set<VariableReferenc
             return newJoinNode;
         }
 
+        private boolean hasNullSkew(JoinNodeStatsEstimate joinEstimate)
```

Comment:
Should this be based on ratio as compared to counts?

Reply:
Yeah but counts are better because irrespective of how big the table is you have to shuffle these anyway so in some sense it's not related to ratio per se